includes/class-core.php

```diff
@@ -87,6 +90,171 @@ public function load_textdomain() {
 		load_plugin_textdomain( 'savage-cards', false, $this->dir . '/languages' );
 	}
 
+	/**
+	 * Register default field group for card meta.
+	 *
+	 * @return void
+	 */
+	public function register_card_meta_field_group() {
+
+		$fields_before = apply_filters( 'savage/card/field_group/fields_before', [] );
+		$fields_after = apply_filters( 'savage/card/field_group/fields_after', [] );
```

Comment:
Is current post type available through this filter? Should we support conditional loading of fields based on current post type?

Reply:
Not available now, but it makes sense to have it. Its not vital to the current NB sprint. Make an issue and add it later?

Reply:
Yup